PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -413,7 +432,7 @@ def produceMET(self, process,  metType, metModuleSequence, postfix):
             process.load("PhysicsTools.PatUtils.patPFMETCorrections_cff")
             
         if postfix != "" and metType == "PF" and not hasattr(process, 'pat'+metType+'Met'+postfix):
-            noClonesTmp = [ "particleFlowDisplacedVertex", "pfCandidateToVertexAssociation" ]
+            noClonesTmp = [ "particleFlowDisplacedVertex", "pfCandidateToVertexAssociation" ] if not self._parameters["Puppi"].value else []
```

Comment:
why was this disabled?
Clones of particleFlowDisplacedVertex and pfCandidateToVertexAssociation can be reintroduced only if they are non-trivial and compute displaced vertex and associations for inputs other than in the original particleFlowDisplacedVertex and pfCandidateToVertexAssociation.
Currently, these are trivial clones.

I removed these earlier because the cost is ~2.5% of reco time already at PU35.